geonode/base/forms.py

```diff
@@ -331,7 +333,11 @@ def build_attrs(self, base_attrs=None, extra_attrs=None, **kwargs):
 
 class ResourceBaseForm(TranslationModelForm):
     """Base form for metadata, should be inherited by childres classes of ResourceBase"""
-
+    abstract = forms.CharField(widget=TinyMCE())
```

Comment:
@petrus7 can't we do this automatically based on the model field (TextField)? Otherwise every time we change the model we will have to keep the form aligned.